mne/io/brainvision/brainvision.py

```diff
@@ -195,7 +195,7 @@ def _read_vmrk_events(fname, event_id=None, response_trig_shift=0):
     for info in items:
         mtype, mdesc, onset, duration = info.split(',')[:4]
         onset = int(onset)
-        duration = int(duration)
+        duration = (int(duration) if duration.isdigit() else 0)
```

Comment:
The safest way would be to return 0 or NaN whenever you can't parse it.

I've only found it in one EEGLAB-exported BV files, which write a plaintext NaN for boundary events. In principle, that field should only have ints in the first place, this is more to deal with a known bad thing EEGLAB does. We can go the safer route, but that may be YAGNI. If it would appear in well-behaved files, Teon probably would have encountered the problem himself.